custom-login/pom.xml

```diff
@@ -63,7 +63,7 @@
         <dependency>
             <groupId>com.nimbusds</groupId>
             <artifactId>nimbus-jose-jwt</artifactId>
-            <version>8.20</version>
+            <version>9.1.2</version>
```

Comment:
This one can be removed now, IIRC it was added as the transitive dependency was old (and was failing the dependency-check plugin)

Reply:
Ignoring this for now (per https://okta.slack.com/archives/DUWA931JT/p1605910458037900).